### PR TITLE
Enable sentry error tracking

### PR DESCRIPTION
**Pull request by krzkaczor**
Closes FRO-661

### CODE REVIEW

File: packages/app/.env.staging [hunk 1]
```diff
@@ -4,9 +4,11 @@ VITE_FEATURE_DEV_SANDBOX=1
...
 VITE_FEATURE_TOS_REQUIRED=1
 VITE_FEATURE_TOP_BANNER=1
+VITE_FEATURE_SENTRY_DSN="https://68f6d4c946de613d425d6c540e4624ec@o4507226038206464.ingest.de.sentry.io/4507226039713872"
 VITE_DEV_ACTIONS_SETTINGS=1
 VITE_DEV_AIRDROP_BADGE=1
 VITE_WALLET_CONNECT_ID='bd1843f2419f5d4c758366c55f9a556c'
 VITE_ANALYTICS_PLAUSIBLE_ID=''
 
 VITE_API_URL=/api
+VITE_ENV_NAME=staging
```
[the remark this answers is not in the document]
Comment (krzkaczor) [reply packages/app/.env.staging:14]: `# human readable name of the environment (e.g. development, staging, production)`

This helps to differentiate between production environment and staging. Is used by sentry but I suspect that can be useful in other scenarios as well.

File: packages/app/vite.config.ts [hunk 2]
```diff
@@ -13,14 +14,20 @@ export default defineConfig({
...
   plugins: [
     react({
       plugins: [['@lingui/swc-plugin', {}]],
     }),
     tsconfigPaths(),
     lingui(),
     svgr(),
+    sentryVitePlugin({
+      org: 'phoenixlabs',
+      project: 'javascript-react',
```
Comment (yivlad) [inline packages/app/vite.config.ts:27]: Should be modified?

File: packages/app/.env.development [hunk 3]
```diff
@@ -7,6 +7,9 @@ VITE_FEATURE_TOP_BANNER=0
 VITE_DEV_ACTIONS_SETTINGS=1
 VITE_DEV_AIRDROP_BADGE=1
 VITE_WALLET_CONNECT_ID='bd1843f2419f5d4c758366c55f9a556c'
+
 VITE_ANALYTICS_PLAUSIBLE_ID=''
+VITE_SENTRY_DSN=''
 
 VITE_API_URL=/api
+VITE_ENV_NAME=development
```
Comment (yivlad) [inline packages/app/.env.development:15]: nit: new lines at the end of files

**Review by oskarvu — APPROVED** [review]

File: packages/app/.env.development (hunk 3 above)
Comment (krzkaczor) [reply packages/app/.env.development:15]: IMO as long as we don't have a formatter doing it one way or the other, i dont care ;d

**Review by yivlad — APPROVED** [review]

**Pull request merged**